assembly/fabric8-stacks/src/main/resources/stacks.json

```diff
@@ -64,6 +64,108 @@
       "mediaType": "image/svg+xml"
     }
   },
+  {
+    "id": "nodejs-postgres",
+    "creator": "Ibrahim Jarif",
+    "name": "Nodejs and Postgres Stack",
+    "description": "Nodejs stack with Postgres",
+    "scope": "advanced",
+    "tags": [
+      "node.js",
+      "npm",
+      "postgres"
+    ],
+    "components": [
+      {
+        "name": "Node.js",
+        "version": "8.11.3"
+      },
+      {
+        "name": "NPM",
+        "version": "5.6.0"
+      },
+      {
+        "name": "Postgres",
+        "version": "9.6"
+      }
+    ],
+    "workspaceConfig": {
+      "defaultEnv": "default",
+      "environments": {
+        "default": {
+          "machines": {
+            "database": {
+              "attributes": {
+                "memoryLimitBytes": "1147483648"
+              },
+              "servers": {
+                "postgres": {
+                  "attributes": {},
+                  "port": "5432",
+                  "protocol": "tcp"
+                }
+              },
+              "volumes": {
+                "postgresdata":{
+                  "path": "/var/lib/pgsql/data"
+                }
+              },
+              "installers": [
+                "org.eclipse.che.exec",
+                "org.eclipse.che.terminal"
+              ],
+              "env": {
+                "POSTGRESQL_ADMIN_PASSWORD": "adminpassword",
+                "POSTGRESQL_DATABASE": "my_data",
+                "POSTGRESQL_USER": "user",
+                "POSTGRESQL_PASSWORD": "password"
+              }
+            },
+            "nodejs": {
+              "attributes": {
+                "memoryLimitBytes": "1147483648"
+              },
+              "servers": {
+                "node": {
+                  "attributes": {},
+                  "port": "8080",
+                  "protocol": "http"
+                }
+              },
+              "volumes": {},
```

Comment:
@ibuziuk @amisevsk Do we need to mount a volume here for the user data? Like `/projects` ?

Reply:
no, project is mounted by default. No need to explicitly specify if